draft-ietf-rats-corim.md

```diff
@@ -86,6 +86,7 @@ normative:
       ITU-T: Recommendation X.690
     target: https://www.itu.int/rec/T-REC-X.690
   IANA.named-information: named-info
+  RFC8152: cose
```

Comment:
we already have an entry for COSE (as STD96), please use that (which also points to the COSEbis document)